compiler/lib/eval.ml

```diff
@@ -84,21 +126,26 @@ let eval_prim x =
   | Ult, [ Int i; Int j ] -> bool (Int32.(j < 0l) || Int32.(i < j))
   | Extern name, l -> (
       let name = Primitive.resolve name in
+      let (module Int : Int) =
+        match Config.target () with
```

Comment:
`Config.target ()` could be called once per call to `f ()`

Reply:
Done

Reply:
It doesn't seem to be committed

Reply:
Sorry. Now it is.